data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -38,34 +41,76 @@
         },
     },
 }
+DATAPROC_JOB = {    # Dataproc job configuration
+    "placement": {"cluster_name": DATAPROC_CLUSTER},
+    "pyspark_job": {
+        "main_python_file_uri": f"gs://{BUCKET_NAME}/{BUCKET_BLOB}",
+        "args": [BQ_TABLE, BUCKET_NAME, "--dry-run"],
+        "jar_file_uris": ["gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"],
+    },
+}
+
+
+@pytest.fixture(autouse=True)
+def setup_and_teardown_table():
+    bq_client = bigquery.Client()
+
+    # Create dataset and load table
+    dataset = bigquery.Dataset(BQ_DATASET)
+    dataset = bq_client.create_dataset(dataset)
+
+    # Load table from dataframe
+    df = pd.read_csv(CSV_FILE)
+    job_config = bigquery.LoadJobConfig(
+        autodetect=True,
+        write_disposition="WRITE_TRUNCATE"
+    )
+    operation = bq_client.load_table_from_dataframe(
+        df, BQ_TABLE, job_config=job_config
+    )
+
+    # Wait for job to complete
+    operation.result()
+
+    yield
+
+    # Delete dataset
+    bq_client.delete_dataset(BQ_DATASET, delete_contents=True)
 
 
 @pytest.fixture(autouse=True)
 def setup_and_teardown_cluster():
-    # Create cluster using cluster client
+    # Create Dataproc cluster using cluster client
     cluster_client = dataproc.ClusterControllerClient(
         client_options={"api_endpoint": f"{CLUSTER_REGION}-dataproc.googleapis.com:443"}
     )
-    operation = cluster_client.create_cluster(PROJECT, CLUSTER_REGION, CLUSTER_DATA)
+    operation = cluster_client.create_cluster(
+        PROJECT_ID, CLUSTER_REGION, CLUSTER_CONFIG
+    )
```

Comment:
There was a [change](https://googleapis.dev/python/dataproc/latest/dataproc_v1/services.html?highlight=create%20cluster#google.cloud.dataproc_v1.services.cluster_controller.ClusterControllerClient.create_cluster) to the API this week, can you please change this to the following?

```
operation = cluster_client.create_cluster(
    project_id=PROJECT_ID, region=REGION, cluster=CLUSTER_CONFIG
)
```